pkgs/by-name/gn/gnome-pass-search-provider/package.nix

```
@@ -0,0 +1,77 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3Packages,
+  wrapGAppsHook3,
+  gtk3,
+  gobject-introspection,
+  gnome,
+}:
+
+let
+  inherit (python3Packages)
+    dbus-python
+    pygobject3
+    fuzzywuzzy
+    levenshtein
+    ;
+in
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gnome-pass-search-provider";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "jle64";
+    repo = "gnome-pass-search-provider";
+    rev = finalAttrs.version;
+    hash = "sha256-PDR8fbDoT8IkHiTopQp0zd4DQg7JlacA6NdKYKYmrWw=";
+  };
+
+  nativeBuildInputs = [
+    python3Packages.wrapPython
+    wrapGAppsHook3
+  ];
+
+  propagatedBuildInputs = [
+    dbus-python
+    pygobject3
+    fuzzywuzzy
+    levenshtein
+
+    gtk3
+    gobject-introspection
+  ];
+
+  env = {
+    LIBDIR = builtins.placeholder "out" + "/lib";
+    DATADIR = builtins.placeholder "out" + "/share";
+  };
+
+  postPatch = ''
+    substituteInPlace  conf/org.gnome.Pass.SearchProvider.service.{dbus,systemd} \
+      --replace-fail "/usr/lib" "$LIBDIR"
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    bash ./install.sh
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    makeWrapperArgs=( "''${gappsWrapperArgs[@]}" )
+    wrapPythonProgramsIn "$out/lib" "$out $propagatedBuildInputs"
+  '';
+
+  meta = {
+    description = "Pass password manager search provider for gnome-shell";
+    homepage = "https://github.com/jle64/gnome-pass-search-provider";
+    license = lib.licenses.gpl3;
+    maintainers = with lib.maintainers; [ lelgenio ];
```

```suggestion
    maintainers = with lib.maintainers; [ lelgenio ];
    mainProgram = "gnome-pass-search-provider";
```
(if that is correct)

---

This a gnome extension, so there's no plain executable.